### PR TITLE
Scrollable clues at tablet breakpoint

### PR DESCRIPTION
**Pull request by desbo**
We've had a few complaints from tablet users about having to scroll up and down too much when reading the list of clues. 

This limits the height of the clues to around the same size as the crossword and makes it scrollable. It also jumps to the appropriate clue (if not already visible) when you click a cell in the grid.

This depends on a change to `scroller` (#10770) to allow an element to be passed in.

oct 05 2015 15 16

### CODE REVIEW

File: static/src/javascripts/es6/projects/common/modules/crosswords/clues.js [hunk 1]
```diff
@@ -41,6 +44,45 @@ class Clue extends React.Component {
...
+    componentDidUpdate (prev) {
+        if (!prev.focussed || prev.focussed.id !== this.props.focussed.id) {
+            this.scrollIntoView(this.props.focussed);
+        }
+    }
+
+    scrollIntoView (clue) {
+        const buffer = 100;
+        const node = React.findDOMNode(this.refs[clue.id]);
+        const visible = node.offsetTop - buffer > this.$cluesNode.scrollTop &&
```
Comment (gtrufitt) [inline static/src/javascripts/es6/projects/common/modules/crosswords/clues.js]: Should we wrap this in the ``detect.getBreakpoint() === 'tablet'`` as accessing offsetTop, scrollTop etc causes reflow and we can avoid doing that on narrow?

Comment (desbo) [inline static/src/javascripts/es6/projects/common/modules/crosswords/clues.js]: good call

**Comment by gtrufitt** [comment]
Looks ace! :+1:  Other than minor question.

**Comment by ScottPainterGNM** [comment]
Is this an option for smaller laptops? We've had a few people complain about similar issues with the clues being alot lower then the grid when doing it on smaller devices that aren't tablets.

**Comment by desbo** [comment]
@ScottPainterGNM it'll work on any device at the tablet breakpoint (between 740 and 980 px)

**Pull request merged**